src/styles/main.scss

```diff
@@ -0,0 +1,138 @@
+@import "./index.scss";
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid $concrete-color;
+  background: $white;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px;
+  }
+
+  &__title {
+    display: flex;
+    width: 166px;
+    flex-direction: column;
+    flex-shrink: 0;
+    margin: 0;
+  }
+
+  &__title-title {
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 8px 0 0;
+    color: $main-accent-color;
+  }
+
+  &__title-code {
+    color: $secondary-color;
+    font-family: $body-font;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    width: 166px;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 8px;
+  }
+
+  &__reviews-counter {
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-accent-color;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 22px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #616070;
+    margin: 0;
+  }
+
+  &__price-sum {
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+    margin: 0;
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $blue-accent-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: $white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    margin: 16px;
+  }
+
+  &__button:hover {
+    background-color: $white;
+    color: $blue-accent-color;
+    border: 1px solid $blue-accent-color;
+    width: 164px;
+    height: 38px;
+  }
+
+}
+
+.stars {
```

Comment:
every block should be in seperate file, create stars.scss file for this block, same with card and also move all styles like body,*,  :root etc to separate file too